### ISSUE TITLE
Contact for detailing

### ISSUE THREAD

**Issue by ymmm-4** [XL-H/ACDC_preprocess#4]
Please add my wechat ID: yahm30. The Synapse dataset has two txt files but in your code it has three.

**Comment by xiaol827** [comment]
@YumnaMemon Hi, the code is used to process ACDC dataset (normalizing and transform it's format from .nii.gz to .npz). For Synapse dataset, please relate [here](https://github.com/Beckschen/TransUNet/blob/main/datasets/README.md)

**Comment by ymmm-4** [comment]
can you send me preprocessed data on my email address pls yumna.memon@gmail.com.

**Comment by xiaol827** [comment]
> can you send me preprocessed data on my email address pls [yumna.memon@gmail.com](mailto:yumna.memon@gmail.com).

Sorry for that, private sharing is not allowed. I recommend you first apply for permission of the dataset and download it, then use this code or other versions for further preprocessing.

**Comment by ymmm-4** [comment]
Can we use processed data from TransUnet. Pls give a look https://drive.google.com/drive/folders/1ACJEoTp-uqfFJ73qS3eUObQh52nGuzCd

**Comment by ymmm-4** [comment]
Error: C:\Users\Administrator\anaconda3\envs\PytorchCuda\python.exe C:/Users/Administrator/PycharmProjects/Swin-Unet-main/train.py --cfg configs/swin_tiny_patch4_window7_224_lite.yaml
=> merge config from configs/swin_tiny_patch4_window7_224_lite.yaml
SwinTransformerSys expand initial----depths:[2, 2, 2, 2];depths_decoder:[1, 2, 2, 2];drop_path_rate:0.2;num_classes:9
---final upsample expand_first---
pretrained_path:./pretrained_ckpt/swin_tiny_patch4_window7_224.pth
---start load pretrained modle of swin encoder---
Namespace(accumulation_steps=None, amp_opt_level='O1', base_lr=0.01, batch_size=24, cache_mode='part', cfg='configs/swin_tiny_patch4_window7_224_lite.yaml', dataset='Synapse', deterministic=1, eval=False, img_size=224, list_dir='.\lists\lists_Synapse', max_epochs=150, max_iterations=30000, n_gpu=1, num_classes=9, opts=None, output_dir='./output', resume=None, root_path='..\data\Synapse\train_npz\data', seed=1234, tag=None, throughput=False, use_checkpoint=False, zip=False)
The length of train set is: 0
0%| | 0/150 [00:00<?, ?it/s]0 iterations per epoch. 0 max iterations
save model to ./output\epoch_99.pth
67%| | 100/150 [00:00<00:00, 250.47it/s]save model to ./output\epoch_149.pth
67%| | 100/150 [00:01<00:00, 84.12it/s]
save model to ./output\epoch_149.pth
